charts/newrelic-infrastructure/templates/ksm/_agent-config_helper.tpl

```diff
@@ -0,0 +1,20 @@
+{{- /*
+Defaults for ksm's agent config
+*/ -}}
+{{- define "nriKubernetes.ksm.agentConfig.defaults" -}}
+is_forward_only: true
+is_secure_forward_only: true
```

Comment:
ditto

Reply:
This conversation is being done [here](https://github.com/newrelic/nri-kubernetes/pull/383#discussion_r844038246).

Resolving.